### DISCUSSION TITLE
Support for FrankenPHP/Caddy Server

### DISCUSSION THREAD

**Discussion by MagePsycho** [wardenenv/warden#741] [Ideas]
People say https://frankenphp.dev/ is quite fast because of Worker mode and many more.
It would be great to test it out with Warden env.

**Comment by bap14** [comment]
Just to add to this, starting with Magento 2.4.7 the GraphQL service can utilize a new Application Server based on OpenSwoole.

https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html?lang=en#application-server-for-graphql-apis

> Commerce 2.4.7-beta2 introduces a new PHP Application Server for GraphQL APIs that is implemented on an Openswoole PHP framework. This application server enables Adobe Commerce to maintain state among Commerce GraphQL API requests and eliminates the need for bootstrapping, which results in higher throughput, lower latency, and efficient resource use for all GraphQL APIs. By sharing application state among processes, GraphQL API requests become significantly more efficient, and GraphQL API responses are on average 30% faster.

Instructions for on-prem Magento installations are here: https://experienceleague.adobe.com/docs/commerce-operations/performance-best-practices/performance-best-practices/application-server.html?lang=en#enable-application-server-on-on-premises-deployments
